core/src/main/kotlin/org/evomaster/core/output/TestCaseWriter.kt

```diff
@@ -650,6 +651,7 @@ class TestCaseWriter {
                         if (printableTh != "null"
                                 && printableTh != NOT_COVERED_YET
                                 && !printableTh.contains("logged")
+                                && !printableTh.contains("""\w+:\d{4,5}""".toRegex())
                         ) {
```

Comment:
you are repeating the same regular expression several times. put it to a final constant variable